app/views/antminer.py

```diff
@@ -34,10 +48,11 @@ def miners():
     hash_rates = {}
     hw_error_rates = {}
     uptimes = {}
-    total_hash_rate_per_model = {"L3+": 0,
-                                 "S7": 0,
-                                 "S9": 0,
-                                 "D3": 0}
+    mapping = {'L3+': 'MH/s', 'S7': 'GH/s', 'S9': 'GH/s', 'D3': 'MH/s'}
```

Comment:
Shall we get rid of this dict and add the values directly into `total_hash_rate_per_model` ?

Reply:
Mind elaborating how you thought on getting rid of the mapping? didn't follow

Reply:
Mind elaborating, not sure if i followed. Its used by both total_hash_rate_per_model

Reply:
I meant something like this:
```
total_hash_rate_per_model = {"L3+": {"value": 0, "unit": "MH/s" },
                             "S7": {"value": 0, "unit": "GH/s" },
                             "S9": {"value": 0, "unit": "GH/s" },
                             "D3": {"value": 0, "unit": "MH/s" }}
```
and this is a suggestion only, I want your opinion on this. Why to add one more variable ?

Reply:
ah, that makes sense, still keeping at least one.